types/validator/index.d.ts

```diff
@@ -417,7 +417,7 @@ declare namespace validator {
     /**
      * Check if the string is a valid date.
      */
-    export function isDate(str: string, options?: IsDateOptions): boolean;
+    export function isDate(input: string | Date, formatOrOptions?: string | IsDateOptions): boolean;
```

Comment:
I think it would be good if `isDate(new Date(), { strictMode: true })` would give a typing error. Do you agree?

Reply:
Repushed with an additional overload. 

Seems to me, it should not be a type error. The js source code will `return false` at the end instead of `throw new Error`.

```typescript
export function isDate(
    input: Date, 
    formatOrOptions: IsDateOptions & { strictMode: true }
): false;
```